scripts/ecg_learning.py

```diff
@@ -0,0 +1,103 @@
+import pdb
+import json
+import requests
+import os
+import argparse
+import findspark
+
+from datetime import datetime, timedelta
+from pyspark.sql import SparkSession
+from pyspark.sql.types import StructField, StructType, StringType, IntegerType
+from pathlib import Path
+from pyspark.sql import functions as F
+
+from utils import create_json, write_data_to_blob, get_data_from_blob
+
+priometheus_host = os.environ['PROMETHEUS_HOST']
+findspark.init()
+
+def parsing_necessary_details(from_time, to_time):
+    url = "{}/prometheus/api/v1/query_range".format(priometheus_host)
+    querystring = {"query": "sum(rate(nginx_request_status_count{cluster=~\"Swarm1|Swarm2\"}[5m]))","start":str(from_time),"end":str(to_time),"step":"900"}
+    headers = {
+        'Accept': "application/json, text/plain, */*",
+    }
+
+    response = requests.request("GET", url, headers=headers, params=querystring)
+
+    return response.json()["data"]["result"][0]["values"]
+
+
+def remove_last_day(df):
+    current_hour = current_time.hour
+    if current_hour == 0 & df.count() >= 672:
```

Comment:
I am assuming that this only deletes the first day data only after the current day is appended.

Reply:
I was deleting the first day's data before merging the old and 1-hour dataframe. In the new commit, I have updated the script for deleting the first-day data only after the current day is appended.